packages/components/inputs/async-select-input/src/async-select-input.tsx

```diff
@@ -362,6 +368,7 @@ const AsyncSelectInput = (props: TAsyncSelectInputProps) => {
           maxMenuHeight={props.maxMenuHeight}
           menuPortalTarget={props.menuPortalTarget}
           menuShouldBlockScroll={props.menuShouldBlockScroll}
+          closeMenuOnSelect={props.closeMenuOnSelect}
```

Comment:
I think it would be great to also add the default value to the `defaultProps` attribute of the component.